packages/server/src/utils/fetch.ts

```diff
@@ -87,21 +87,6 @@ export async function _fetch(
   });
 
   if (response && response.status >= 200 && response.status < 300) {
-    // const _res = response.clone();
```

Comment:
@scottydawg I did this to help with the error coming back from the server, but since it's so random, it's probably more worth while to fix however this is happening there, vs a permanent bandaid